stats/stats/src/charts/lines/average_gas_price.rs

```diff
@@ -31,7 +34,7 @@ const GWEI: i64 = 1_000_000_000;
 pub struct AverageGasPriceStatement;
 
 impl StatementFromRange for AverageGasPriceStatement {
-    fn get_statement(range: Option<Range<DateTimeUtc>>) -> Statement {
+    fn get_statement(range: Option<Range<DateTimeUtc>>, _: &BlockscoutMigrations) -> Statement {
         sql_with_range_filter_opt!(
             DbBackend::Postgres,
             r#"
```

Comment:
It is needed to be checked (e.g., via running `explain analyze`), but I may suggest that adding additional `transactions.block_timestamp != to_timestamp(0) AND transactions.block_consensus = true` into where condition may improve the speed of the query because some of transactions would be eliminated before the join. 

It may also help to apply similar logic to range filter condition, but I haven't completely get how the generation of range filter statement works

Reply:
Explain on a real data has shown very minor cost increase with the added filter. It seems that removing ~0.2% of rows yields less benefit than checking the condition on each transaction. (It was already scanned sequentially, so the only difference is added condition)

Reply:
also tried adding `{filter}` to the transactions - the results got much worse 🫠

Reply:
Wait, it seems it was an anomaly. Now with `filter` for `transactions` it's ~3-4 times faster

Reply:
also it makes sense according to explain, it now uses indexes instead of sequential scan

Reply:
tested on staging sepolia - `explain` with and without filter in txns shows barely any difference in planning. The only differences are:
- added that filter on `Seq scan` on table `transactions`.
- changed order of sort/aggregate at the end
It seems that it makes sense to filter as much txns as possible here.

Reply:
Tried executing with everything included. For current query metrics show >30 mins of execution for 1 day update + executing it by hand already took >1 minute (with only consensus & 0 timestamp filter). So I assume the metric is correct.

Now, prepare, executing it with added general timestamp filter took (!) 15 SECONDS. OK this is huge !!!